run/events-pubsub/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Google LLC. All rights reserved.
+# Use of this source code is governed by the Apache 2.0
+# license that can be found in the LICENSE file.
+
+# [START run_pubsub_dockerfile]
```

Comment:
remove duplicated region tag

Reply:
Well, won't we get includecode errors then?
I've changed to `run_events_pubsub_dockerfile`.